### PR TITLE
Added Get Options Section using SDK

### PR DESCRIPTION
**Pull request by AirTechWick**
I finished adding a section that shows how to create a component using the SDK in React Native, but I still haven't shown how to add that component to the demo app. I'll update it showing how to add the component to the Demo.

### CODE REVIEW

File: docs/react-native-theta-client-1_2.md [hunk 1]
```diff
@@ -25,6 +25,112 @@ yarn run android
...
+const GetOptions = ({ navigation }) => {
+  const [optionNames, setOptionNames] = useState([]);
+
+  useEffect(() => {
+    const fetchOptions = async () => {
+      const result = await getOptions([
+        OptionNameEnum.CameraControlSource,
+        OptionNameEnum.CameraMode,
+        OptionNameEnum.PowerSaving,
+        OptionNameEnum.CaptureMode,
```
Comment (jcasman) [inline docs/react-native-theta-client-1_2.md:74]: If these are the 4 options that you pick, as examples, please explain that somewhere. There's a long list of options they could include.

Also, please make sure the output image matches the code.

File: docs/images/react_native/windows/getoptionscrn.png
Comment (jcasman) [inline docs/images/react_native/windows/getoptionscrn.png:1]: This output does not seem to match the code. Please check if you are using an older image or older code example.

**Review by jcasman — COMMENTED** [review]
Please address comments I have added. Add a sentence saying why you picked the options you picked. (It's fine if it's just for an example.) And please make sure the output display matches the code.

File: docs/react-native-theta-client-1_2.md (hunk 1 above)
Comment (AirTechWick) [reply docs/react-native-theta-client-1_2.md]: I added new commits that are more consistent with the options

File: docs/images/react_native/windows/getoptionscrn.png
Comment (AirTechWick) [reply docs/images/react_native/windows/getoptionscrn.png:1]: Output should match the code now

**Comment by AirTechWick** [comment]
Issues mentioned above were adressed in new updates

**Comment by codetricity** [comment]
theta-client 1.3 is out.  https://github.com/ricohapi/theta-client/releases/tag/1.3.0

**Review by jcasman — APPROVED** [review]

**Comment by jcasman** [comment]
> theta-client 1.3 is out. https://github.com/ricohapi/theta-client/releases/tag/1.3.0

@AirTechWick As we discussed this morning, please look see if getThetaModel() is a new command. Also please test _bitrate, _burstMode. Please create a post for the community on what you find.

**Comment by codetricity** [comment]
The `_bitrate` options are new.

**Comment by codetricity** [comment]
Did you see the API documentation for theta-client?

https://ricohapi.github.io/theta-client/

**Pull request merged**